src/core/qgsfeature.h

```diff
@@ -598,6 +598,8 @@ class CORE_EXPORT QgsFeature
       sipRes = sipConvertFromNewType( v, sipType_QVariant, Py_None );
     }
     % End
+#else
+    QVariant attribute( const QString &name ) const;
```

Comment:
well is there a way to do `#if...not...def SIP_RUN` otherwise, we would have to use something like `#ifdef NOTSIP`and set `NOTSIP` to `TRUE`

I agree it would be more readable to have SIP afterwards.